components/installation-telemetry/pkg/server/installationAdmin.go

```diff
@@ -17,12 +17,17 @@ type InstallationAdminSettings struct {
 	SendTelemetry bool `json:"sendTelemetry"`
 }
 
-type InstallationAdminData struct {
+type Data struct {
+	InstallationAdmin InstallationAdmin `json:"installationAdmin"`
```

Comment:
Adding the new attribute `installationAdmin` also means that the data structure in Segment will change, correct? Then we would probably better copy ID and Settings to the Data struct before we send the data. Otherwise, it would break analytics with the changing data structure.